pkg/util/labels.go

```diff
@@ -0,0 +1,35 @@
+package util
+
+import (
+	"github.com/cespare/xxhash/v2"
+	"github.com/prometheus/prometheus/model/labels"
+)
+
+var seps = []byte{'\xff'}
+
+// StableHash is a labels hashing implementation which is guaranteed to not change over time.
+// This function should be used whenever labels hashing backward compatibility must be guaranteed.
+func StableHash(ls labels.Labels) uint64 {
```

Comment:
Can't find use of the function. If we need it, should we maybe place it in the `model` package?

Reply:
We'll use it for the compactor yes ! I can move it to model.